src/openfermion/utils/_operator_utils.py

```diff
@@ -839,9 +849,36 @@ def normal_ordered(operator, hbar=1.):
         order_fn = normal_ordered_quad_term
         kwargs['hbar'] = hbar
 
+    elif isinstance(operator, InteractionOperator):
```

Comment:
What does it mean to normal-order an InteractionOperator?

Reply:
The docstring for `normal_ordered` says that, in addition to putting creation operators before annihilation operators, it orders the operators in each set by index. `InteractionOperator` takes care of the former by construction; this does the latter.

Reply:
This actually orders from lowest to highest, so I'll need to either update the docstring or change the code. Why the convention that the modes are ordered from highest to lowest?

Reply:
I don't know why that convention was picked. In any case, let's stick with the old convention.

Reply:
Done

Reply:
@kevinsung ready for re-review